docs/octo-usage.md

```diff
@@ -0,0 +1,72 @@
+# k8s octo usage with multiple JVBs
+
+## Context
+
+### Application context
+Jitsi-meet is an opensource WebRTC application devided in several services which can be hosted separately. The Jitsi VideoBridge (JVB) service is a selective forwarding unit used for audio and video redistribution among all participants.
+​
+JVBs are scalable so that conferences and participants can be connected to different JVBs. If Octo is not used, one conference has all its participants in the same JVB, but multiple conferences can be on that JVB. This creates a problem as JVBs have limited resources : once a conference is started on a JVB, it is bound to it. If too many people join the conference, the JVB will be overloaded and experience will be poor.
```

Comment:
```suggestion
JVBs are scalable so that conferences and participants can be connected to different JVBs. If Octo is not used, one conference has all its participants in the same JVB, but multiple conferences can be on that JVB. This creates a problem as JVBs have limited resources: once a conference is started on a JVB, it is bound to it. If too many people join the conference, the JVB will be overloaded and experience will be poor.
```